web/templates/settings/generate_integration_url_modal.hbs

```diff
@@ -26,6 +26,23 @@
         <label for="integration-url-topic-input">{{t "Topic"}}</label>
         <input type="text" id="integration-url-topic-input" class="modal_text_input integration-url-parameter" maxlength="{{ max_topic_length }}" />
     </div>
+    <div id="filter-event-checkbox" class="input-group hide">
+        <label class="checkbox">
+            <input type="checkbox" id="show-event-types"/>
+            <span></span>
+        </label>
+        <label class="inline" for="show-event-types">
+            {{t "Filter events that will trigger notifications?"}}
+        </label>
+    </div>
+    <div class=" input-group hide" id="checkbox-ui">
+        <label for="events-box">{{t "Events to include:"}}</label>
```

Comment:
The id/class names here are not great as I can't read them in the js code and know what they are referring to.

When I read `checkbox_ui`, I'm wondering which checkboxes as there are a bunch? For what setting - topic/events? UI where? `events-box` again I don't know what this is when I read it.

I'd remove "checkbox" or "box" from all the classes/ids you're creating. Make sure "integration" is in the class/id. And think about how the class or id reads in the js code and let that help your naming.

For example, "events-box" might read better in the js code as something like "integration-events-container" or "individual-integration-events-container".

Reply:
See updated review comments for more naming suggestions.